Program.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Pomelo.EntityFrameworkCore.MySql.Infrastructure;
+using System;
+using System.Text;
+using Newtonsoft.Json;
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddDbContext<ApplicationDbContext>(options =>
+{
+    var connectionString = "Server=localhost;Database=myappdb;User=myappuser;Password=mypassword;";
+    Console.WriteLine($"Connection String: {connectionString}");
+    options.UseMySql(connectionString, ServerVersion.AutoDetect(connectionString));
+});
+
+// Move this line before builder.Build();
+builder.Services.AddHealthChecks();
+
+var app = builder.Build();
+
+app.UseHttpsRedirection();
+app.UseRouting();
+
+app.UseEndpoints(endpoints =>

```

Comment:
Mapping endpoints is way simpler than this. it is as simple as `app.MapGet` or similar.

Check the docs: https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.builder.endpointroutebuilderextensions.mapget?view=aspnetcore-7.0